tests/lms/extensions/feature_flags/__init___test.py

```diff
@@ -0,0 +1,53 @@
+from unittest import mock
+
+import pytest
+from pyramid.interfaces import IRequestExtensions
+
+from lms.extensions.feature_flags import includeme
+
+
+class TestIncludeMe:
+    def test_it_creates_the_feature_flags_object(self, pyramid_config, FeatureFlags):
+        includeme(pyramid_config)
+
+        FeatureFlags.assert_called_once_with()
+
+    def test_feature(self, feature_flags, pyramid_config, pyramid_request):
+        includeme(pyramid_config)
+        feature = pyramid_config.registry.queryUtility(IRequestExtensions).methods[
```

Comment:
I think this test would be more useful if it were actually able to call `pyramid_request.feature("test_feature")`, in the way that actual application code would.

Reply:
Indeed, but I couldn't figure out how to do that :man_shrugging: What it actually does is functionally equivalent to that (the `feature` it extracts is the exact function that would be `request.feature()`) it just has to extract it in this awkward way

Reply:
We have code in h that does this using `pyramid.request.apply_request_extensions`: https://github.com/hypothesis/h/blob/8132480cd750e4ce1c1760e9b807b6b9b02135b4/tests/h/services/feature_test.py#L43